src/videodownlinkbandwidthpolicy/VideoPriorityBasedPolicy.ts

```diff
@@ -275,7 +274,7 @@ export default class VideoPriorityBasedPolicy implements VideoDownlinkBandwidthP
   protected calculateOptimalReceiveStreams(): void {
     const chosenStreams: VideoStreamDescription[] = [];
     const remoteInfos: VideoStreamDescription[] = this.videoIndex.remoteStreamDescriptions();
-    if (remoteInfos.length === 0 || !this.videoPreferences || this.videoPreferences.isEmpty()) {
+    if (remoteInfos.length === 0 || this.videoPreferences?.isEmpty()) {
```

Comment:
Checking, what happens when `this.videoPreferences` is null? Earlier it would go and set `this.optimalReceiveStreams` to `[]` but now it wont go in the `if` right?

Reply:
it should never be null and just be undefined. Previously it never go in because we always init it to an empty set.